documentation/book/security.adoc

```diff
@@ -31,31 +41,31 @@ NOTE: "certificates rotation" for generating new ones on their expiration will b
 
 The data encryption is provided on two different listeners exposed by each Kafka broker.
 
-Interbroker communication is through the `REPLICATION` listener on port 9091, which is encrypted by default.
+Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
 Encrypted communication with clients is provided through the `CLIENTTLS` listener on port 9093.
 
 NOTE: Un-encrypted communication with clients is still possible through the `CLIENT` listener on port 9092.
 
 === Clients connection via TLS
 
-If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the client's CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
-The clients CA certificate can be extracted from the generated `<cluster-name>-kafka-cert` Secret with the following command if the Kafka cluster is running on {OpenShiftName}.
+If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the cluster CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
+The cluster CA certificate can be extracted from the generated `<cluster-name>-cluster-ca-cert` Secret with the following command if the Kafka cluster is running on {OpenShiftName}.
```

Comment:
This sentence would read a bit strangely in productized docs (what else would the Kafka cluster be running on?). Also elsewhere in the docs we put the k8s description first and the OS description 2nd. I think we should move towards generating different docs for each platform, because that's the only way we can properly deal with this sort of thing. Not for this PR, obvs.

Reply:
I had already a chat about this with the docs team. They also didn't liked the way we have the Kubernetes and OpenShift sections. Hopefully they will have some good solution.